miso-web/src/main/webapp/scripts/runLibraryMetrics_ajax.js

```diff
@@ -65,11 +65,21 @@ var RunLibraryMetrics = (function($) {
         Utils.showOkDialog('Error', ['No changes to save']);
         return;
       }
-      Utils.ajaxWithDialog('Saving...', 'PUT', Urls.rest.qcStatuses.bulkUpdate, changes.map(Utils.array.get('update')), function() {
-        changes.forEach(function(change) {
-          change.rowData.update(change.update);
+      function save() {
+        Utils.ajaxWithDialog('Saving...', 'PUT', Urls.rest.qcStatuses.bulkUpdate, changes.map(Utils.array.get('update')), function() {
+          changes.forEach(function(change) {
+            change.rowData.dataReview = null; // TODO: test
```

Comment:
Ticket for TODO?

Reply:
This was more a note to self that I forgot to remove. Removed now